CommandHelper/LocalPackages/commands/buffs/deathstormraymondrusch.ms

```diff
@@ -0,0 +1,38 @@
+@plr = player()
+
+
+@hero = associative_array(
+            'name': '§0Bl§ba§7ck§8-§bLa§7nt§0er§bn§8-§8R§bR',
+            'class': 'blacklanterndeathstormraymondrusch',
```

Comment:
Raymond Rusch? I can't find anyone called Raymond Rusch? Did you mean Raymond Stein?